services/ui-auth/AccountId.js

```diff
@@ -0,0 +1,4 @@
+// AccountId.js
```

Comment:
this file would ideally be located in a top level 'libs' directory.  The virtue of putting it there is that all services could reference the single file, instead of having a copy in each service.